src/ctapipe/reco/impact.py

```diff
@@ -85,51 +90,59 @@ class ImPACTReconstructor(Component):
 
     """
 
+    minimiser = traits.CaselessStrEnum(
+        ["minuit", "l-BFGS"],
+        default_value="minuit",
+        help="name minimiser to use in the fit",
+    ).tag(config=True)
+
+    use_time_gradient = traits.Bool(
+        default_value=False, help="Use time gradient in ImPACT reconstruction"
+    ).tag(config=True)
+
+    root_dir = traits.Unicode(
+        default_value=".", help="Directory containing ImPACT tables"
+    ).tag(config=True)
+
     # For likelihood calculation we need the with of the
     # pedestal distribution for each pixel
     # currently this is not available from the calibration,
     # so for now lets hard code it in a dict
     ped_table = {
-        "LSTCam": 2.8,
-        "NectarCam": 2.3,
-        "FlashCam": 2.3,
+        "LSTCam": 1.4,
+        "NectarCam": 1.3,
+        "FlashCam": 1.3,
+        "SST-Camera": 0.5,
         "CHEC": 0.5,
-        "DUMMY": 0,
+        "ASTRICam": 0.5,
+        "dummy": 0.01,
+        "UNKNOWN-960PX": 1.0,
     }
-    spe = 0.5  # Also hard code single p.e. distribution width
+    spe = 0.6  # Also hard code single p.e. distribution width
+
+    property = ReconstructionProperty.ENERGY | ReconstructionProperty.GEOMETRY
 
     def __init__(
-        self,
-        root_dir=".",
-        minimiser="minuit",
-        prior="",
-        template_scale=1.0,
-        xmax_offset=0,
-        use_time_gradient=False,
+        self, subarray, atmosphere_profile, dummy_reconstructor=False, **kwargs
     ):
         """
         Create a new instance of ImPACTReconstructor
         """
```

Comment:
Missing documentation for the parameters.

Reply:
The docstring should be completely removed and the parameters of `__init__` documented in the class docstring